src/util/redis/redisConnector.js

```diff
@@ -101,6 +101,7 @@ const RedisDB = {
       stats.increment("redis_error", {
         operation: "get"
       });
+      log.error(`Error getting value from Redis: ${e}`);
```

Comment:
isn't this is logged in versioned router?

Reply:
No we are catching this error at destination level only